include/riak_kv_types.hrl

```diff
@@ -34,5 +34,8 @@
 
 -define(DATATYPE_STATS_DEFAULTS, [actor_count]).
 
+-define(R1_DATATYPE_VERSIONS, [{?COUNTER_TYPE, 2}]).
```

Comment:
Could we add some comments above this? Even R1/R2 isn't immediately obvious: is it 1.4 vs 2.0 or 2.0.<4 vs 2.0.>4?